docs/source/models/concept_models/vivarium_mncnh_portfolio/concept_model.rst

```diff
@@ -594,6 +594,10 @@ Default stratifications to all observers should include scenario and input draw.
       * ANC attendance
       * Ultrasound coverage
     - NOT YET INCLUDED
+  * - 8. Received probiotics counts
+    - * Gestational age (above/below 37 weeks)
+      * Delivery facility type
+    - NOT YET INCLUDED
```

Comment:
```suggestion
```

It looks like we already have a probiotics observer! So I made some suggested changes to remove this new one and then request to have the existing one stratified by preterm status in the table below.